worker.py

```diff
@@ -16,6 +16,13 @@
 import constants
 from exceptions import WorkerUnableToExecuteTaskError
 
+class UniqueIDGenerator:
+    def __init__(self):
+        self.counter = 0
+
+    def get_unique_id(self):
+        self.counter += 1
```

Comment:
Would this guarantee uniqueness in counter if multiple threads are calling get_unique_id at the same time?
Guid guarantees that.